website/docs/learn/1-setting-up.md

```diff
@@ -0,0 +1,120 @@
+
+If you are already **experienced with dbt**, you still need to complete this tutorial as we'll be building on this project in our lessons.
+
+
+
+<LoomVideo id="cb99861ab1034f7fab5fa48529e61f85" />
+
+<Alert type="info">
+Please note that this tutorial is adapted from the main <a href="https://tutorial.getdbt.com/tutorial/setting-up/">Getting Started</a> tutorial — there may be some references to BigQuery in various videos, however the same concepts should apply on Snowflake.
+</Alert>
+
+This tutorial is based on a fictional business named `jaffle_shop`, so you'll see this name used throughout the project.
+
+In this tutorial, we will be turning the following query into a dbt project that is tested and documented, ready to be deployed.
+```sql
+with customers as (
+
+    select
+        id as customer_id,
+        first_name,
+        last_name
+
+    from raw.jaffle_shop.customers
+
+),
+
+orders as (
+
+    select
+        id as order_id,
+        user_id as customer_id,
+        order_date,
+        status
+
+    from raw.jaffle_shop.orders
+
+),
+
+customer_orders as (
+
+    select
+        customer_id,
+
+        min(order_date) as first_order_date,
+        max(order_date) as most_recent_order_date,
+        count(order_id) as number_of_orders
+
+    from orders
+
+    group by 1
+
+),
+
+
+final as (
+
+    select
+        customers.customer_id,
+        customers.first_name,
+        customers.last_name,
+        customer_orders.first_order_date,
+        customer_orders.most_recent_order_date,
+        coalesce(customer_orders.number_of_orders, 0) as number_of_orders
+
+    from customers
+
+    left join customer_orders using (customer_id)
+
+)
+
+select * from final
+```
+
+## Connect to Snowflake
+For dbt Learn, we've created a Snowflake account that you will be added to, but dbt works with [many data warehouses](https://docs.getdbt.com/docs/supported-databases).
+
+1. Go to the Snowflake Console (link provided in email) and login with your supplied credentials.
+2. Reset your password: all students will be asked to update their password — please use a password manager to store this securely.
+3. Copy and paste the above query into a worksheet to validate that you are able to run it successfully.
+<Lightbox src="/img/successful-snowflake-query.png" title="Ensure you can run the above query" />
+
+
+### FAQs
+<FAQ src="faqs/loading-data" alt_header="The data in this tutorial is already loaded into Snowflake. How do I load data into my warehouse?" />
+<FAQ src="faqs/database-privileges" />
+
+
+## Create a repository
+We're going to use [GitHub](https://github.com/) as our git provider for this tutorial, but you can use any git provider. If you don't yet have a GitHub account, [create one now](https://github.com/join).
+<LoomVideo id="afe148aeab5e4279a2ca310251ea20a6" />
+
+1. Create a new GitHub repository [here](https://github.com/new) named `dbt-learn-[initiallastname]` (e.g. `dbt-learn-ccarroll`).
```

Comment:
```suggestion
1. Create a new GitHub repository [here](https://github.com/new) named `dbt-learn-[initialsurname]` (e.g. `dbt-learn-ccarroll`).
```
Later on you call this `initialsurname`, which I prefer, I think it's easier to read that way